### PR TITLE
company extraction code

### PR DESCRIPTION
**Pull request by HungUnicorn**
Would all file paths must be imported from conf.properties and Config?

### CODE REVIEW

File: analysis/src/main/java/io/ssc/trackthetrackers/analysis/statistics/CompanyAggregatedCentrality.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+import org.apache.flink.api.java.ExecutionEnvironment;
+import org.apache.flink.api.java.aggregation.Aggregations;
+import org.apache.flink.api.java.operators.DataSource;
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.api.java.tuple.Tuple3;
+import org.apache.flink.core.fs.FileSystem.WriteMode;
+import org.apache.flink.util.Collector;
+
+// Aggregate Centrality to company level
+public class CompanyAggregatedCentrality {
```
Comment (sscdotopen) [inline analysis/src/main/java/io/ssc/trackthetrackers/analysis/statistics/CompanyAggregatedCentrality.java]: please use 2 spaces indentation

File: analysis/src/main/java/io/ssc/trackthetrackers/analysis/statistics/ThirdPartyAggregatedCentrality.java [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+import org.apache.flink.api.java.aggregation.Aggregations;
+import org.apache.flink.api.java.operators.DataSource;
+import org.apache.flink.api.java.tuple.Tuple1;
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.api.java.tuple.Tuple3;
+import org.apache.flink.core.fs.FileSystem.WriteMode;
+import org.apache.flink.util.Collector;
+
+// Get the aggregated Centrality for each third party, and get the top K third party.
+public class ThirdPartyAggregatedCentrality {
```
Comment (sscdotopen) [inline analysis/src/main/java/io/ssc/trackthetrackers/analysis/statistics/ThirdPartyAggregatedCentrality.java]: again, please use 2 spaces indentation

**Comment by sscdotopen** [comment]
Hi Hung,

I'm not sure about this pull request. Maybe this code should only reside in your repository, as it seems to be very specific to your thesis. What do you think?

**Comment by HungUnicorn** [comment]
I think the same.

**Pull request closed without merging**